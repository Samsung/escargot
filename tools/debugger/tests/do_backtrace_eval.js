eval("function f() {\nreturn 5\n}")
f()

function g(x) {
  return x
}

eval("var x = 0\ng()\ng()")

function h() {
  var g = 1
  eval("g = 2\ng = 3")
  global_eval = eval
  global_eval("x=1\n\ng()")
}

h()
