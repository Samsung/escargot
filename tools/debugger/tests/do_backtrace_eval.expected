Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_backtrace_eval.js:1
(escargot-debugger) s
Stopped at eval code:1
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:2
(escargot-debugger) s
Stopped at eval code:2 (in f() at line:1, col:1)
(escargot-debugger) bt
eval code:2:1 [depth:0] (in f)
tools/debugger/tests/do_backtrace_eval.js:2:1 [depth:1]
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:8
(escargot-debugger) s
Stopped at eval code:1
(escargot-debugger) s
Stopped at eval code:2
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:5 (in g() at line:4, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_backtrace_eval.js:5:3 [depth:0] (in g)
eval code:2:1 [depth:1]
tools/debugger/tests/do_backtrace_eval.js:8:1 [depth:2]
(escargot-debugger) s
Stopped at eval code:3
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:5 (in g() at line:4, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_backtrace_eval.js:5:3 [depth:0] (in g)
eval code:3:1 [depth:1]
tools/debugger/tests/do_backtrace_eval.js:8:1 [depth:2]
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:17
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:11 (in h() at line:10, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:12 (in h() at line:10, col:1)
(escargot-debugger) s
Stopped at eval code:1
(escargot-debugger) s
Stopped at eval code:2
(escargot-debugger) bt
eval code:2:1 [depth:0]
tools/debugger/tests/do_backtrace_eval.js:12:3 [depth:1] (in h)
tools/debugger/tests/do_backtrace_eval.js:17:1 [depth:2]
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:13 (in h() at line:10, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:14 (in h() at line:10, col:1)
(escargot-debugger) s
Stopped at eval code:1
(escargot-debugger) s
Stopped at eval code:3
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace_eval.js:5 (in g() at line:4, col:1)
(escargot-debugger) bt
tools/debugger/tests/do_backtrace_eval.js:5:3 [depth:0] (in g)
eval code:3:1 [depth:1]
(escargot-debugger) c
Connection closed.
